### PR TITLE
Added Stats Page Logic using CodeU Guidelines

### PR DESCRIPTION
**Pull request by azanbinzahid**

### CODE REVIEW

**Review by waahm7 — COMMENTED** [review]
Great work!, Looks Good.

File: .factorypath [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<factorypath>
```
Comment (raufrajar) [inline .factorypath]: This file is specific to your setup. Such files should not go into the repo.
You can make git ignore certain files by adding them into .gitignore (https://git-scm.com/docs/gitignore). Would recommend adding this to .gitignore and update the pull request.

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -83,6 +84,15 @@ public void storeMessage(Message message) {
     return messages;
   }
 
+// <<<<<<< statspage-azan
```
Comment (raufrajar) [inline src/main/java/com/google/codeu/data/Datastore.java]: nit: seems leftover from merge, you can remove this comment and the one below.

**Review by raufrajar — CHANGES_REQUESTED** [review]
Please update .gitignore

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 2 above)
Comment (azanbinzahid) [reply src/main/java/com/google/codeu/data/Datastore.java]: Removed. I thought we can keep it for tracking changes later.

File: .factorypath (hunk 1 above)
Comment (azanbinzahid) [reply .factorypath]: updated.

**Review by raufrajar — APPROVED** [review]
LGTM!

**Pull request merged**